integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/logsearch/UrlParameters.java

```diff
@@ -0,0 +1,122 @@
+package com.sequenceiq.it.cloudbreak.newway.logsearch;
+
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Joiner;
+import com.sequenceiq.it.cloudbreak.CredentialTests;
+
+public class UrlParameters {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CredentialTests.class);
+
+    private String timeRangeType = "LAST";
+
+    private String timeRangeUnit = "h";
+
+    private String timeRangeInterval = "72";
+
+    private String sortingKey = "logTime";
+
+    private String sortingType = "desc";
+
+    private Integer pageSize = 100;
+
+    private Integer page = 0;
+
+    private String query;
+
+    private UrlParameters(String query) {
+        this.query = query;
+    }
+
+    public static Map<String, Object> createParametersMap(List<QueryModel> queryModels, ObjectMapper objectMapper) {
+        try {
+            UrlParameters urlParameters = new UrlParameters(objectMapper.writeValueAsString(queryModels));
+            return objectMapper.convertValue(urlParameters, new TypeReference<Map<String, Object>>() {
+            });
+        } catch (JsonProcessingException e) {
+            LOGGER.error("Error during json convert:", e);
+            return null;
```

Comment:
Considering to return an empty map instead of null? Or maybe Optional of this kind of Map